docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
-        <h3>CCG Standards Stack</h3>
-        <img src="resources/standards-stack.png" alt="standards-stack" width="100%">
+        <h2>Presentations</h2>
         <p>
-          The above diagram illustrates how traceability-interop fits into the
-          larger family of W3C CCG standards.
+          Presentations in this spec are
+          <a
+            href="https://w3c-ccg.github.io/traceability-vocab/#TraceablePresentation">Traceable
+            Presentations</a>, which contain several notable identifiers
+          that can be used for correlation, retrieval, and business rule
+          processing of data.
         </p>
+        These identifiers are:
         <ul>
           <li>
-            At the <em>Business</em> layer, supply chain actors are exchanging
-            data in support of exchanged goods and services.
-          </li>
-          <li>
-            The <em>Trust</em> layer is supported by actors signing and
-            verifying data as Verifiable Credentials.
-          </li>
-          <li>
-            The <em>Process</em> defined standardized APIs for how to exchange
-            Verifiable Credentials (VC-API) and Verifiable Presentations
-            (trace-interop).
+            `TraceablePresentation.id` — The unique ID of the
+            presentation itself; think of this as identifying the wire
+            transaction. This ID MUST be a UUID v4 per [[rfc4122]].
           </li>
           <li>
-            <em>Linked Data</em> establishes concise semantic context of the
-            data exchanged, enabled with JSON-LD by the trace-vocab
-            specification.
+            `TraceablePresentation.workflow.definition` — The ID for the
+            workflow definition, or the <em>type</em> of workflow that
+            the presented credentials belong to. This ID MUST be a UUID
+            v4 per [[rfc4122]].
           </li>
           <li>
-            <em>Semantics</em> are defined in a multitude of established
-            vocabularies and come in many forms, from narrowly specialized,
-            industry-vertical vocabularies, to broadly-applicable, generic
-            vocabularies.
+            `TraceablePresentation.workflow.instance` — The ID for the
+            unique instance of the workflow, e.g., party A sending goods
+            of a certain type (with particular data requirements to meet
+            regulatory needs) to party B using workflow type X as
+            defined by `TraceablePresentation.workflow.definition`. This
+            ID MUST be a UUID v4 per [[rfc4122]].
           </li>
         </ul>
-      </section>
+        <p>
+          `TraceablePresentation.id` <em>MUST</em> be unique to each
+          presentation. A presentation received with a duplicate ID
+          <em>MUST</em> be rejected with a 409 Conflict status code per
+          [[rfc9110]]. A presenter encountering this error,
+          <em>SHOULD</em> generate a new presentation, with a new UUID
```

Comment:
sneaky change in a large PR, file an issue :)

Reply:
Added #502

Reply:
can you point to where in this PR you added the #502 issue ?

Reply:
@OR13 I've added a link to #502 below.